inst/starter_models/sir/README.md

```diff
@@ -27,19 +27,6 @@ The size of the total population is,  $ N = S + I + R$.
 
 
 ```mermaid
-%%{
-  init: {
-    'theme': 'base',
-    'themeVariables': {
-      'primaryColor': '#FDBF57',
-      'primaryTextColor': '#000',
-      'primaryBorderColor': '#5E6A71',
-      'lineColor': '#7A003C',
-      'secondaryColor': '#fff',
-      'tertiaryColor': '#fff'
-    }
-  }
-}%%
 flowchart LR;
```

Comment:
I Think the issue is with these greek letters that do not seem to be rendering on github. I recommend waiting on diagrams for now ... we can do something better and more automated.